### ISSUE TITLE
[qt5-webview] build failure

### ISSUE THREAD

**Issue by z-ninja** [microsoft/vcpkg#7353]
**Host Environment**
- OS: Windows 7
- Compiler: Visual studio 2015

**To Reproduce**
vcpkg.exe install qt5-webview:x86-windows

**Failure logs** 
The following packages will be built and installed:
  * qt5-base[core]:x86-windows
  * qt5-declarative[core]:x86-windows
  * qt5-modularscripts[core]:x86-windows
    qt5-webview[core]:x86-windows
Additional packages (*) will be modified to complete this operation.
Starting package 1/4: qt5-base:x86-windows
Building package qt5-base[core]:x86-windows...
-- Using cached E:/v/vcpkg/downloads/qtbase-everywhere-src-5.12.3.tar.xz
-- Using source at E:/v/vcpkg/buildtrees/qt5-base/src/5.12.3-4fa71cacd1
-- Configuring x86-windows-dbg
-- Configuring x86-windows-dbg done
-- Configuring x86-windows-rel
-- Configuring x86-windows-rel done
-- Package x86-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:58 (message):
    Command failed: E:/v/vcpkg/downloads/tools/jom/jom-1.1.3/jom.exe /J 2
    Working Directory: E:/v/vcpkg/buildtrees/qt5-base/x86-windows-dbg
    Error code: 2
    See logs for more information:
      E:\v\vcpkg\buildtrees\qt5-base\build-x86-windows-dbg-out.log
      E:\v\vcpkg\buildtrees\qt5-base\build-x86-windows-dbg-err.log

Call Stack (most recent call first):
  ports/qt5-base/install_qt.cmake:48 (vcpkg_execute_required_process)
  ports/qt5-base/portfile.cmake:151 (install_qt)
  scripts/ports.cmake:74 (include)

Error: Building package qt5-base:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: qt5-base:x86-windows
  Vcpkg version: 2019.07.18-nohash

Additionally, attach any relevant sections from the log files above.
build-x86-windows-dbg-err.log file
`
jom 1.1.3 - empower your cores

cl : Command line warning D9025 : overriding '/Fo./' with '/Foqlibraryinfo_final.obj'
Project MESSAGE: Enabling D3D11 mode for ANGLE
jom: E:\v\vcpkg\buildtrees\qt5-base\x86-windows-dbg\src\angle\src\compiler\preprocessor\Makefile.Debug [.obj\debug\DiagnosticsBase.obj] Error 2
jom: E:\v\vcpkg\buildtrees\qt5-base\x86-windows-dbg\src\angle\src\compiler\preprocessor\Makefile.Debug [.obj\debug\Macro.obj] Error 2
jom: E:\v\vcpkg\buildtrees\qt5-base\x86-windows-dbg\src\angle\src\compiler\preprocessor\Makefile [debug] Error 2
jom: E:\v\vcpkg\buildtrees\qt5-base\x86-windows-dbg\src\angle\src\compiler\Makefile [sub-preprocessor-make_first-ordered] Error 2
jom: E:\v\vcpkg\buildtrees\qt5-base\x86-windows-dbg\src\angle\src\Makefile [sub-compiler-make_first-ordered] Error 2
jom: E:\v\vcpkg\buildtrees\qt5-base\x86-windows-dbg\src\angle\Makefile [sub-src-make_first] Error 2
jom: E:\v\vcpkg\buildtrees\qt5-base\x86-windows-dbg\src\Makefile [sub-angle-make_first] Error 2
jom: E:\v\vcpkg\buildtrees\qt5-base\x86-windows-dbg\Makefile [sub-src-make_first] Error 2
`
**Additional context**
Hi, i need help to make this work.

**Comment by z-ninja** [comment]
My mistake again.
Suggestion:You should try to include some proximity of how much space is needed for library to build like others package managers shows download and install size of package will be consumed.
Beside that, you are building best package manager for windows I have seen so far.
Best regards.
Reason of failure build: not enough disk space.

**Comment by forscienceo** [comment]
> My mistake again.
> Suggestion:You should try to include some proximity of how much space is needed for library to build like others package managers shows download and install size of package will be consumed.
> Beside that, you are building best package manager for windows I have seen so far.
> Best regards.
> Reason of failure build: not enough disk space.

i have same problem but i have enough space for this.

**Comment by brainplot** [comment]
@forscienceo I was also getting the same error, even though I had plenty of disk space.
The problem was that I cloned `vcpkg` in a directory that was too deep in my filesystem and some packages fail to build because paths get very long.
I cloned `vcpkg` under `C:\` and then it built successfully. That's likely the problem, even though the directory doesn't really seem too long to you. I initially cloned in `C:\Users\$USER\Projects` and that was still too long.
